workflows/metatdenovo.nf

```diff
@@ -68,7 +68,7 @@ include { DIGINORM } from '../subworkflows/local/diginorm'
 
 include { PROKKA_CAT   } from '../subworkflows/local/prokka_cat'
 include { TRANSDECODER } from '../subworkflows/local/transdecoder'
-
+include { SUB_EUKULELE } from '../subworkflows/local/folder_eu'
```

Comment:
Why is the subworkflow called `folder_eu`? And why can't I see that file here? Did you forget to include it in the commit?